addons/website/static/tests/tours/link_tools.js

```diff
@@ -371,7 +408,8 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     {
         content: "Verify that the link label input does not contain ZWS",
         trigger: "#o_link_dialog_label_input:value('Contact Us')",
+        extra_trigger: "div#oe_snippets:not(div.o_we_ui_loading)",
         isCheck: true,
     },
-    ...wTourUtils.clickOnSave(),
+    ...wTourUtils.clickOnSave("bottom", 20000),
```

Comment:
Is this tour particularly slow? We generally tend to never increase timeout without a very good explanation of why, I am surprise to see it lost in the diff

Reply:
The end of this tour is particularly slow yes. I've try with 10000 and didn't work on localhost.